src/content/link.tsx

```diff
@@ -65,6 +65,10 @@ export const link = {
     WCAG_4_1_2: guidanceLinkTo('WCAG 4.1.2', 'https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html'),
     WCAG_4_1_3: guidanceLinkTo('WCAG 4.1.3', 'https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html', [guidanceTags.WCAG_2_1]),
     BingoBakery: linkTo('Bingo Bakery Video', 'https://go.microsoft.com/fwlink/?linkid=2080372'),
+    VoiceControlVideo: linkTo('Voice Control Video', 'https://www.youtube.com/watch?v=bAFzL-dOzu0'),
```

Comment:
Should these be fwlinks? Youtube video links feel disproportionately likely to be the sort of thing that we might want to be able to change quickly in the future

Reply:
I considered it, but we havent changed the BingoBakery since we created it,and this ones are also unlikely to change, so decided against it